client/src/app/game/board/board.tsx

```diff
@@ -121,6 +118,9 @@ function Board({onClick, localGameState}: Props) {
 		)
 	}
 
+	const noAvailableActions = () =>
```

Comment:
Please define this function before it is used.

Reply:
Moved